ios/FluentUI/Other Cells/ActionsCell.swift

```diff
@@ -15,51 +15,85 @@ import UIKit
         case regular
         case destructive
         case communication
 
-        func highlightedTextColor(for window: UIWindow) -> UIColor {
+        func highlightedTextColor(tokens: ActionsCellTokens) -> UIColor {
             switch self {
             case .regular:
-                return Colors.primary(for: window).withAlphaComponent(0.4)
+                return UIColor(dynamicColor: tokens.regularTextColor).withAlphaComponent(0.4)
             case .destructive:
-                return Colors.Table.ActionCell.textDestructiveHighlighted
+                return UIColor(dynamicColor: tokens.destructiveTextColor).withAlphaComponent(0.4)
             case .communication:
-                return Colors.Table.ActionCell.textCommunicationHighlighted
+                return UIColor(dynamicColor: tokens.communicationTextColor).withAlphaComponent(0.4)
             }
         }
 
-        func textColor(for window: UIWindow) -> UIColor {
+        func textColor(tokens: ActionsCellTokens) -> UIColor {
             switch self {
             case .regular:
-                return Colors.primary(for: window)
+                return UIColor(dynamicColor: tokens.regularTextColor)
             case .destructive:
-                return Colors.Table.ActionCell.textDestructive
+                return UIColor(dynamicColor: tokens.destructiveTextColor)
             case .communication:
-                return Colors.Table.ActionCell.textCommunication
+                return UIColor(dynamicColor: tokens.communicationTextColor)
             }
         }
     }
 
     public static let identifier: String = "ActionsCell"
 
-    @objc public class func height(action1Title: String, action2Title: String = "", containerWidth: CGFloat) -> CGFloat {
+    // MARK: - ActionsCell TokenizedControl
+    @objc public var actionsCellOverrideTokens: ActionsCellTokens? {
+        didSet {
+            self.overrideTokens = actionsCellOverrideTokens
+        }
+    }
+
+    let defaultTokens: ActionsCellTokens = .init()
+    var tokens: ActionsCellTokens = .init()
+    /// Design token set for this control, to use in place of the control's default Fluent tokens.
+    var overrideTokens: ActionsCellTokens? {
+        didSet {
+            updateTokens()
+        }
+    }
+
+    public func overrideTokens(_ tokens: ActionsCellTokens?) -> Self {
+        overrideTokens = tokens
+        return self
+    }
+
+    @objc private func themeDidChange(_ notification: Notification) {
+        guard let window = window, window.isEqual(notification.object) else {
+            return
+        }
+        updateTokens()
+    }
+
+    private func updateTokens() {
+        tokens = resolvedTokens
+        updateActionTitleColors()
+        backgroundColor = UIColor(dynamicColor: tokens.cellBackgroundColor)
+    }
+
+    @objc public class func height(action1Title: String, action2Title: String = "", containerWidth: CGFloat, tokens: ActionsCellTokens) -> CGFloat {
```

Comment:
Do you need to pass in the tokens here? I think you should be able to just use the cell's tokens.

Reply:
Ah. `class func`. Never mind.